README.md

```diff
@@ -270,7 +270,8 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
                 },
 
                 // The path to the TLS certificates directory.
-                // Used for HTTPS connections.
+                // Used for client HTTPS connections to the OpenID Provider.
+                // Typically, this should point to a standard certificate directory such as "/etc/ssl/cert".
```

Comment:
This line needs to be broken across two lines to avoid the need for the horizontal scrollbar.